tests/optimisation_test/optimization.config.yaml

```diff
@@ -0,0 +1,40 @@
+experiment:
+  components:  # list of optimization features and bounds
+    P10-MIX1_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    water_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    NaCl-3M_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    PVP-1wt_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    SDS-1wt_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    Sodiumsilicate-1wt_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    AcidRed871_0gL_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    RhodamineB1_0gL_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    MethyleneB_250mgL_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    NaOH-1M_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+    L-Cysteine-100gL_to_dispense:
+      upper_bound: 5
+      lower_bound: 0
+optimizer:
```

Comment:
as per my previous comment, you can add an extra field here with any additional args/parameters the user would want to pass to the optimiser constructor